Tools/clinic/clinic.py

```diff
@@ -221,6 +221,23 @@ def c_repr(s: str) -> str:
     return '"' + s + '"'
 
 
+def wrapped_c_string_literal(
+        text: str,
+        *,
+        width: int = 72,
+        suffix: str = '',
+        initial_indent: int = 0,
+        subsequent_indent: int = 4
+) -> str:
+    wrapped = textwrap.wrap(text, width=width, replace_whitespace=False,
+                            drop_whitespace=False, break_on_hyphens=False)
+    suffix += "\n"
+    lines = [f'"{line}"{suffix}' for line in wrapped]
+    separator = subsequent_indent * ' '
+    joined = initial_indent * ' ' + separator.join(lines)
+    return joined.removesuffix(suffix)
```

Comment:
It can be written in two lines as
```suggestion
    separator = '"' + suffix + '\n' + subsequent_indent * ' ' + '"'
    return initial_indent * ' ' + '"' + separator.join(wrapped) + '"'
```
or
```suggestion
    separator = f'"{suffix}\n{subsequent_indent * " "}"'
    return f'{initial_indent * " "}"{separator.join(wrapped)}"'
```

Reply:
Indeed it can; that's even neater.

Reply:
I prefer the first variant because of better syntax highlighting.

Reply:
`{initial_indent * " "}` in f-string can also be written as `{"":{initial_indent}}`, but it may be too cryptic.

Reply:
Yes that's too cryptic.